yaksh/templates/yaksh/download_questionpaper.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
```

Comment:
This causes the footer of the preview page and thus the footer of the printed page to have a "Developed by fossee" footer note. I am not sure this is necessary.